src/pkg/services/m365/api/mail.go

```diff
@@ -406,6 +407,24 @@ func (c Mail) GetItem(
 			ByAttachmentId(ptr.Val(a.GetId())).
 			Get(ctx, attachConfig)
 		if err != nil {
+			if graph.IsErrCannotOpenFileAttachment(err) {
+				logger.CtxErr(ctx, err).With("skipped_reason", fault.SkipNotFound).Info("attachment not found")
+				el.AddSkip(
+					fault.AttachmentSkip(
+						fault.SkipNotFound,
+						itemID,
```

Comment:
This is going to be a problem.  Fault items deduplicate [based on the item id](https://github.com/alcionai/corso/blob/main/src/pkg/fault/item.go#L83).  This skip would be read as "we are skipping the entire item", when you mean for it to indicate that we're skipping just the attachment within that item.  If you can use the attachment ID instead, and add the item ID into the extended map, that would solve the issue.

Also, please coordinate with @juleslasarte about the sdk expectations here.  The introduction of subcomponent skips is a new paradigm, and other frameworks need to handle it gracefully.  As a v1, it might be smarter for us to not include a fault skip at all, and instead log and continue.  That will unblock the fix, and we can investigate the fault skip records as a follow-up.

Reply:
> Also, please coordinate with @juleslasarte about the sdk expectations here. The introduction of subcomponent skips is a new paradigm, and other frameworks need to handle it gracefully. As a v1, it might be smarter for us to not include a fault skip at all, and instead log and continue. That will unblock the fix, and we can investigate the fault skip records as a follow-up.

I second this -- to simply skip for now, and add the fault records as a follow up so we don't need to coordinate on the work on action details and this.

Reply:
I've created https://github.com/alcionai/corso/issues/3620 to track this